xml/System.Security.Cryptography/RSACng.xml

```diff
@@ -32,6 +32,9 @@
 -   The key used by <xref:System.Security.Cryptography.RSACng> is managed by a separate <xref:System.Security.Cryptography.CngKey> object. In contrast, <xref:System.Security.Cryptography.RSACryptoServiceProvider> has a key that is directly tied to the operations of the type itself.  
   
 -   <xref:System.Security.Cryptography.RSACng> performs such operations as signing and verifying signatures by using properties of the <xref:System.Security.Cryptography.RSACng> object, just as <xref:System.Security.Cryptography.ECDsaCng> uses its object properties to control signing and verification operations.  
+
+> [!NOTE]
+> The <xref:System.Security.Cryptography.RSACng> class is a CNG implementation for Windows platforms. In a cross-platform scenario, you should use the platform agnostic <xref:System.Security.Cryptography.RSA.Create%2A?displayProperty=fullName> to create an instance of the default implementation for the platform where your code will run.
```

Comment:
> RSACng isn't "a CNG implementation"

Ref: https://docs.microsoft.com/en-us/dotnet/api/system.security.cryptography.rsacng?view=netframework-4.7

> Provides a Cryptography Next Generation (CNG) implementation of the RSA algorithm.

reads ... "a ... CNG ... implementation ..."

... along with the note from @joshfree ...

> developers should be using RSA.Create (platform agnostic) instead of directly using the concrete RSACng (windows) or RSAOpenSsl (linux, mac) classes

"RSACng (windows)" ... led to ... "... for Windows".

RSA.Create was also floated there, so that's what made the note as the suggestion for xplat development.

Of course, I can make it say whatever you want. Do you want to give me the note language, and I'll shoot it right in there?